Resources/doc/reference/configuration/headers.rst

```diff
@@ -245,6 +249,12 @@ This enables a simplistic ETag calculated as md5 hash of the response body:
     page. If the page depends on permissions, make sure to make the ETag differ
     based on those permissions (e.g. by appending the :doc:`user context hash </features/user-context>`).
 
+    You should want to set your ETags weak when you are using gzip compression on webserver.
```

Comment:
the spelling does not like webserver. i think we opted for the more grammatically correct "web server" elsewhere in the doc. can you please adjust that?

Reply:
and etags is also not found in spelling. can we say "You want to set weak ETag headers when ..."